### PR TITLE
Added route to deactivate customers account

### PR DESCRIPTION
**Pull request by mm12063**

### CODE REVIEW

**Comment by mm12063** [comment]
Looking into the codecov decerement

File: tests/test_routes.py [hunk 1]
```diff
@@ -299,6 +298,17 @@ def test_update_customer_not_found(self):
         resp = self.app.put(f"{BASE_URL}/0", json={"not": "today"})
         self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
 
+<<<<<<< HEAD
```
Comment (TestSubjector) [inline tests/test_routes.py]: There is remnants from merging, you need to remove this line.

File: tests/test_routes.py [hunk 2]
```diff
@@ -299,6 +298,17 @@ def test_update_customer_not_found(self):
...
+    def test_deactivate_customer_account(self):
+        """It should deactivate a customer's account"""
+        customer = CustomerFactory()
+        logging.debug(customer)
+        resp = self.app.post(
+            BASE_URL, json=customer.serialize(), content_type="application/json"
+        )
+
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED, "Account created")
+=======
```
Comment (TestSubjector) [inline tests/test_routes.py]: Remove this line

File: tests/test_routes.py [hunk 3]
```diff
@@ -314,10 +324,33 @@ def test_get_customers_by_lastname(self):
         )
 
         self.assertEqual(resp.status_code, status.HTTP_201_CREATED, "Account not created")
+>>>>>>> master
```
Comment (TestSubjector) [inline tests/test_routes.py]: Same here

File: tests/test_routes.py [hunk 4]
```diff
@@ -314,10 +324,33 @@ def test_get_customers_by_lastname(self):
...
         # Make sure location header is set
         location = resp.headers.get("Location", None)
         self.assertIsNotNone(location)
 
+<<<<<<< HEAD
```
Comment (TestSubjector) [inline tests/test_routes.py]: And here

File: tests/test_routes.py [hunk 5]
```diff
@@ -314,10 +324,33 @@ def test_get_customers_by_lastname(self):
...
+        self.assertEqual(new_customer["id"], updated_customer["id"])
+        self.assertFalse(updated_customer["acc_active"])
+
+    def test_deactivate_customer_account_not_found(self):
+        """It should not Get a Customer thats not found in deactivate customer"""
+        resp = self.app.delete(
+            f"{BASE_URL}/0/active", content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+=======
```
Comment (TestSubjector) [inline tests/test_routes.py]: One here

File: tests/test_routes.py [hunk 6]
```diff
@@ -341,3 +374,4 @@ def test_get_customers_by_lastname(self):
         # check the data just to be sure
         for customer in data:
             self.assertEqual(customer["lastname"], lastname_to_use)
+>>>>>>> master
```
Comment (TestSubjector) [inline tests/test_routes.py]: Last one

**Review by TestSubjector — COMMENTED** [review]
Some lines of git merge comment code got added to the files. They need to be removed.

File: tests/test_routes.py (hunk 6 above)
Comment (mm12063) [reply tests/test_routes.py]: This is due to a merge conflict being pushed into master

**Comment by TestSubjector** [comment]
Perfect, LGTM!

**Pull request merged**